### PR TITLE
Add timeout to Network Finality API #383

### PR DESCRIPTION
**Pull request by adecaro**
Signed-off-by: Angelo De Caro <adc@zurich.ibm.com>

### CODE REVIEW

**Comment by adecaro** [comment]
Linked to https://github.com/hyperledger-labs/fabric-smart-client/pulls

File: token/services/network/driver/network.go [hunk 1]
```diff
@@ -58,7 +59,7 @@ type Network interface {
 	IsFinalForParties(id string, endpoints ...view.Identity) error
 
 	// IsFinal takes in input a transaction id and waits for its confirmation.
```
Comment (HagarMeir) [inline token/services/network/driver/network.go:61]: please update comment

File: token/services/ttx/ordering.go [hunk 2]
```diff
@@ -20,14 +22,14 @@ type orderingView struct {
...
-// 1. It broadcasts the token transaction to the proper Fabric ordering service.
+// 1. It broadcasts the token transaction to the proper backend.
 func NewOrderingView(tx *Transaction) *orderingView {
 	return &orderingView{tx: tx}
 }
 
 // Call execute the view.
 // The view does the following:
-// 1. It broadcasts the token token transaction to the proper Fabric ordering service.
+// 1. It broadcasts the token token transaction to the proper backend.
```
Comment (HagarMeir) [inline token/services/ttx/ordering.go:32]: `token token`

**Comment by HagarMeir** [comment]
Do you need to first merge this: https://github.com/hyperledger-labs/fabric-smart-client/pull/388
And only after merging to update `go.mod`?

**Comment by adecaro** [comment]
@HagarMeir , yes, indeed

**Comment by adecaro** [comment]
@HagarMeir , I updated the FSC dep and addressed your comments :)

**Review by HagarMeir — APPROVED** [review]

**Pull request merged**